torch/csrc/jit/runtime/register_ops_utils.cpp

```diff
@@ -428,9 +428,14 @@ void listMulIntRight(Stack* stack) {
 }
 
 void listSlice(Stack* stack) {
-  int64_t step = pop(stack).to<int64_t>();
-  int64_t end = pop(stack).to<int64_t>();
-  int64_t start = pop(stack).to<int64_t>();
+  auto step_val = pop(stack);
+  auto end_val = pop(stack);
+  auto start_val = pop(stack);
+
+  int64_t step = step_val.isInt() ? step_val.to<int64_t>() : 1;
```

Comment:
why is step 1 and not INT64_MAX ? (this may be right just want to double check)

Reply:
this was initially "int step = 1" in the slice schema, but i think you are right. We should probably make it INT64_MAX to be consistent.

Reply:
When there is no step size is given, i think the python assumption is that step size is 1. Because if you do x[1:3], it outputs all elements starting from index 1 to index 3 with one increment.